src/runner/driver-test-runner.js

```diff
@@ -92,8 +92,9 @@ export class DriverTestRunner {
    *
    * @param {string} settings - "browseMode" "focusMode" for NVDA, "pcCursor" "virtualCursor"
    *                 for JAWS., "defaultMode" for others.
+   * @param {string[]} additionalSettings - e.g. "speechRateIncrease" and "commentAnnouncementOn" for JAWS
    */
-  async ensureSettings(settings) {
+  async ensureSettings(settings, additionalSettings) {
```

Comment:
We can revert on including `additionalSettings`, following my comment in https://github.com/w3c/aria-at/pull/1125#discussion_r1939608816. Instead, `settings` will come in as a space separated string to be handled.

Examples of how this has been handled at https://github.com/w3c/aria-at/pull/1126:
* [Alert review page showing that the JAWS command has an additional `speechRateIncrease` and `commentAnnouncementOn` setting. Also added a new `Enter + virtualCursor + commentAnnouncementOn` command](https://deploy-preview-1126--aria-at.netlify.app/review/alert)
* [Alert test page for JAWS commands accounting for multiple settings](https://deploy-preview-1126--aria-at.netlify.app/tests/alert/test-03-triggeralert?at=jaws)